qemu_v8.mk

```diff
@@ -417,6 +417,11 @@ $(ROOTFS_UGZ): u-boot buildroot | $(BINARIES_PATH)
 ################################################################################
 
 XEN_CONFIGS = .config $(ROOT)/build/kconfigs/xen.conf
+ifneq ($(filter 1 2 3,$(SPMC_AT_EL)),)
+XEN_FFA = y
+else
+XEN_FFA = n
+endif
```

Comment:
Is the else part really helpful?
Does `XEN_FFA` needs to be set here since it is already set in `common.mk`?

Reply:
No, I guess not. I'll remove the `common.mk` part instead since we so far are only looking at the the SPMC_AT_EL in `qemu_v8.mk`.